index.js

```diff
@@ -11,68 +11,145 @@ module.exports = {
+    'jest',
+    'unicorn',
+    'vue'
   ],
   settings: {
     'import/resolver': {
       node: { extensions: ['.js', '.mjs'] }
     }
   },
   rules: {
+
+    /**********************/
+    /* General Code Rules */
+    /**********************/
+
     // Enforce import order
-    'import/order': 2,
+    'import/order': 'error',
 
     // Imports should come first
-    'import/first': 2,
+    'import/first': 'error',
 
     // Other import rules
-    'import/no-mutable-exports': 2,
+    'import/no-mutable-exports': 'error',
 
     // Allow unresolved imports
-    'import/no-unresolved': 0,
+    'import/no-unresolved': 'off',
 
     // Allow paren-less arrow functions only when there's no braces
-    'arrow-parens': [2, 'as-needed', { requireForBlockBody: true }],
+    'arrow-parens': ['error', 'as-needed', { requireForBlockBody: true }],
 
     // Allow async-await
-    'generator-star-spacing': 0,
+    'generator-star-spacing': 'off',
 
     // Allow debugger during development
-    'no-debugger': process.env.NODE_ENV === 'production' ? 2 : 0,
-    'no-console': process.env.NODE_ENV === 'production' ? 2 : 0,
+    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
+    'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
 
     // Prefer const over let
-    'prefer-const': [2, {
+    'prefer-const': ['error', {
       'destructuring': 'any',
       'ignoreReadBeforeAssign': false
     }],
 
     // No single if in an "else" block
-    'no-lonely-if': 2,
+    'no-lonely-if': 'error',
 
     // Force curly braces for control flow,
     // including if blocks with a single statement
-    curly: [2, 'all'],
+    curly: ['error', 'all'],
 
     // No async function without await
-    'require-await': 2,
+    'require-await': 'error',
 
     // Force dot notation when possible
-    'dot-notation': 2,
+    'dot-notation': 'error',
 
-    'no-var': 2,
+    'no-var': 'error',
 
     // Force object shorthand where possible
-    'object-shorthand': 2,
+    'object-shorthand': 'error',
 
     // No useless destructuring/importing/exporting renames
-    'no-useless-rename': 2,
+    'no-useless-rename': 'error',
+
+    /**********************/
+    /*   Unicorn Rules    */
+    /**********************/
+
+    // Pass error message when throwing errors
+    'unicorn/error-message': 'error',
+
+    // Uppercase regex escapes
+    'unicorn/escape-case': 'error',
+
+    // Use "new" for most builtins (not String, Number, ..., though)
+    'unicorn/new-for-builtins': 'error',
+
+    // Array.isArray instead of instanceof
+    'unicorn/no-array-instanceof': 'error',
+
+    // for-of instead traditional for loop
+    'unicorn/no-for-loop': 'error',
+
+    // Prevent deprecated `new Buffer()`
+    'unicorn/no-new-buffer': 'error',
+
+    // Keep regex literals safe!
+    'unicorn/no-unsafe-regex': 'off',
+
+    // Use non-fraction numbers if possible, e.g. 1 instead of 1.0
+    'unicorn/no-zero-fractions': 'error',
```

Comment:
I dont think this is a useful rule, although it might be the same for JS adding fractions at least allows you to specify eg intended number precision. So imo it can have value to use `1.00` instead of `1` so future developers know it was intended to be a float with 2 digit precision

Reply:
If I'd need a specific precision, I'd personally not specify that in, say the default value of the function. As this has to be expressed in the function anyway (via `toFixed(2)` for example and has to be documented somehow (code comment, decent fn name if possible), I didn't saw a value in having `1.00` over `1`.

But on the other hand, this rule doesn't "save" a lot on any side, so I have no problem with removing it :relaxed: 

Would like to hear some more opinions though :)

Reply:
I really dont think its worth that much discussion, if you want to enable this just do it. As you already mentioned, we can always change it in our own config.

The main background for my reasoning is that scientifically speaking your precision is only as high as the number with the smallest precision. Which means that if you would list a number as `1` its just plainly wrong and inaccurate to suddenly start using `toFixed(2)` to get a 2 digit precision.
But another reason is that although Javascript might treat them exactly the same, in other programming languages mixing integers with floats could give unexpected results. And this is probably another discussion but I just dont think encouraging javascript specific (bad) behaviour is really worthwhile as a whole for us developers. It only makes transitioning between programming languages harder.

Anyway, as said feel free to do what you think is best :)